src/intervals/gff3/reader.jl

```diff
@@ -9,63 +11,57 @@ Create a reader for data in GFF3 format.
 * `save_directives=false`: if true, store directive lines, which can be accessed
   with the `directives` function
 """
-type GFF3Reader <: Bio.IO.AbstractReader
-    state::Ragel.State
-    version::VersionNumber
-    sequence_regions::Vector{Interval{Void}}
-    key::StringField
+type Reader <: Bio.IO.AbstractReader
+    state::Bio.Ragel.State
```

Comment:
Didn't you change the Ragel stuff to be called something like Automa Helper? Does that need to be done here?